### DISCUSSION TITLE
I didn't get any module

### DISCUSSION THREAD

**Discussion by AgroMeteorology** [pycaret/pycaret#3159] [Q&A]
Sir we run with pycaret with my data for Classification Module it was successfully showed but in best = compare_models() i didn't get any models plz help me in this. 
![pycaret](https://user-images.githubusercontent.com/120000060/206099390-8d147e86-7a82-4128-bf2b-59f961420e1f.PNG)

**Comment by turkalpmd** [comment]
This module actually very full of [arguments](https://pycaret.readthedocs.io/en/stable/api/classification.html);


default selected arguments are like there;


```pycaret.classification.compare_models(include: Optional[List[Union[str, Any]]] = None, exclude: Optional[List[str]] = None, fold: Optional[Union[int, Any]] = None, round: int = 4, cross_validation: bool = True, sort: str = 'Accuracy', n_select: int = 1, budget_time: Optional[float] = None, turbo: bool = True, errors: str = 'ignore', fit_kwargs: Optional[dict] = None, groups: Optional[Union[str, Any]] = None, experiment_custom_tags: Optional[Dict[str, Any]] = None, probability_threshold: Optional[float] = None, verbose: bool = True, display: Optional[Display] = None, parallel: Optional[ParallelBackend] = None)→ Union[Any, List[Any]]```                                      
                                      

If you define `erros='raise'` maybe you can see what is the error. Commonly errors about your data don't learn enough or you have very biased or imbalanced data end model gets overfitted.
